ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent { label "docker-gpu"}
+    environment {
+        DOCKER_BUILDKIT=1
+        UBUNTU_BASE_TAG=20.04
+    }
+    stages {
+        stage("Capture Build Info"){
+            steps {
+                rtBuildInfo (captureEnv: true)
+            }
+        }
+        stage("Build Docker Image") {
+            steps {
+                sh script: """
+                    docker build \
+                        --progress=plain \
+                        --tag pynvml:${BUILD_NUMBER} \
```

Comment:
Tagging docker image to prevent running the wrong container in the next step. Probably an edge case that will not happen ofter, or at all, but it's better to be safe.